test/target_mvd_web/controllers/api/v1/target_controller_test.exs

```diff
@@ -78,6 +78,14 @@ defmodule TargetMvdWeb.API.V1.TargetControllerTest do
       conn = post(conn, Routes.api_v1_target_path(conn, :create), target: @invalid_attrs)
       assert json_response(conn, 422)["errors"] != %{}
     end
+
+    test "renders errors when target limit reached", %{conn: conn, user: user} do
+      1..10
+      |> Enum.each(fn _ -> create_target(%{user: user}) end)
+
+      conn = post(conn, Routes.api_v1_target_path(conn, :create), target: @create_attrs)
+      assert json_response(conn, 422)["errors"] != %{}
```

Comment:
We should check for the error as well when it's implemented

Reply:
Autogen doesn't think so💁